cmake/projects/Lua/hunter.cmake

```diff
@@ -0,0 +1,23 @@
+if(DEFINED HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Lua
+    VERSION
+    "5.3.2"
+    URL
+    "https://github.com/hunter-packages/lua/archive/5.3.2.tar.gz"
```

Comment:
Please apply my Linux fixes by using this code:

```cmake
URL
"https://github.com/hunter-packages/lua/archive/v5.3.2-p0.tar.gz"
SHA1
39fc2bb5e7de0c9b4dd3940ccf99cef3b7cb4b04
```

You can leave version `5.3.2` or update it to `5.3.2-p0` to show that package is patched and not the original.